### DISCUSSION TITLE
[Bug report] After update to 0.46.0 window title bar has gone

### DISCUSSION THREAD

**Discussion by mave-git** [GetPublii/Publii#1609] [Bug Report]
### Operating system

Ubuntu 22.04

### Publii version

0.46.0 (build 16889)

### Issue type

Application

### Bug description

After update to 0.46.0 window title bar has gone.
![Bildschirmfoto vom 2024-08-16 08-14-59](https://github.com/user-attachments/assets/afbc3eef-0776-4ee5-ae67-98e726ed6a2b)

**Comment by GoneFishinJax** [comment]
Seeing the same issue on MX Linux 23.3 | Debian 12.6 | Xfce desktop 4.18.1

Alt+F4 to close

**Comment by dziudek** [comment]
Hi @GoneFishinJax @mave-git 

It will be fixed soon - it seems that since Electron v.30 some window options started to work on Linux, causing this issue :) I did not saw this on my test environment, because I built it under WSL

**Reply by mave-git** [reply]
Ah, this behavior in 0.46.0 seemed unintended ....
Meanwhile I saw is under Ubuntu: Frame / title bar appears in Gnome-Panel, resizing etc. goes via SUPERKEY or Alt + Space.
But a real title bar would be better!

**Reply by GoneFishinJax** [reply]
Thanks, and keep up the great work!

**Comment by bobmitro** [comment]
Hey, 

I've just uploaded a new version ([build 16890](https://getpublii.com/download/)) that should fix the issue with the top bar. Please update the app and let us know if the problem is resolved.
